vita3k/gui/src/main_menubar.cpp

```diff
@@ -26,10 +26,21 @@
 namespace gui {
 
 static void draw_file_menu(GuiState &gui, EmuEnvState &emuenv) {
+    const auto textures_path{ emuenv.shared_path / "textures" };
+
     auto &lang = gui.lang.main_menubar.file;
     if (ImGui::BeginMenu(lang["title"].c_str())) {
         if (ImGui::MenuItem(lang["open_pref_path"].c_str()))
             open_path(emuenv.pref_path.string());
+        if (ImGui::MenuItem(lang["open_textures_path"].c_str())) {
+            if (!fs::exists(textures_path)) {
+                fs::create_directories(textures_path);
+                fs::create_directories(textures_path / "export");
+                fs::create_directories(textures_path / "import");
+                open_path(textures_path.string());
+            } else
+                open_path(textures_path.string());
```

Comment:
move this `open_path` and the one above outside the if exists check to get rid of duplicated code

Reply:
done